cmd/errors/errors.go

```diff
@@ -27,8 +27,9 @@ func AlreadyLogged(err error) bool {
 
 var (
 	// Errors related to package file name
-	ErrBadPackName = errors.New("bad pack name: it must be either a PackID:  packVendor::Pack[@version|@~version|@^version|@>=version] or a local Pack file: <path>/Vendor.Pack.version.pack or a local Pack Description file: <path>/Vendor.Pack.pdsc")
-	ErrBadPackURL  = errors.New("bad pack url: the url provided for this pack is malformed")
+	ErrBadPackName    = errors.New("bad pack name: it must be either a PackID:  packVendor::Pack[@version|@~version|@^version|@>=version] or a local Pack file: <path>/Vendor.Pack.version.pack or a local Pack Description file: <path>/Vendor.Pack.pdsc")
+	ErrBadPackURL     = errors.New("bad pack url: the url provided for this pack is malformed")
```

Comment:
it would be good if could specify what we are expecting.
`https://<something>/packVendor.Pack.version.pack`